app.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+
+var express = require('express');
+var fs = Promise.promisifyAll(require('fs'));
```

Comment:
Actually, I was relying on `service-runner`'s globalisation of bluebird, but now that that's gone, I'll change the code accordingly.